internal/services/monitor/monitor_private_link_scope_resource.go

```diff
@@ -47,6 +48,28 @@ func resourceMonitorPrivateLinkScope() *pluginsdk.Resource {
 				ValidateFunc: validate.PrivateLinkScopeName,
 			},
 
+			"ingestion_access_mode": {
+				Type:     pluginsdk.TypeString,
+				Required: true,
+				ValidateFunc: validation.StringInSlice(
+					[]string{
+						string(privatelinkscopesapis.AccessModeOpen),
+						string(privatelinkscopesapis.AccessModePrivateOnly),
+					},
+					false),
+			},
+
+			"query_access_mode": {
+				Type:     pluginsdk.TypeString,
+				Required: true,
+				ValidateFunc: validation.StringInSlice(
```

Comment:
adding these is a rbeaking change? shouldn't they be optional?

Reply:
@katbyte , good catch. 

My most recent push on this PR has changes to make both ingestion_access_mode and query_access_mode optional with default values of 'Open'. Would you mine reviewing again, please?